app/setup.py

```diff
@@ -0,0 +1,37 @@
+from flask import Flask, render_template, flash
```

Comment:
Why is there an app/setup.py and web/app/setup.py? Contents seem kind of similar